code/game/machinery/suit_storage_unit.dm

```diff
@@ -337,10 +340,13 @@
 		return
 	if(panel_open)
 		if(istype(I, /obj/item/crowbar))
+			if(occupant || helmet || suit || storage || boots)
+				to_chat(user, "<span class='warning'>There are contents that prevent you form deconstructing [src]!</span>")
+				return
 			if(locked)
 				to_chat(user, "<span class='warning'>The security system prevents you from deconstructing [src]!</span>")
 				return
-			dump_contents()
+			dump_contents() // probably still a good idea for just incase?
```

Comment:
Is it? There is another situation where this might be necessary?

Reply:
honestly probably not.. but there might be some edge cases where you put something into it *as* it'd get deconstructed and it just evaporates it.

Reply:
Yeah, seems like a safe thing to do just in case. Wouldn't be the first time SSUs have snapped items out of existence on destroy.